mw-webapp/src/component/modal/PromptModalContent.module.scss

```diff
@@ -1,5 +1,12 @@
 @import "src/styles/variables";
 
 .buttons {
-  margin-top: $marginMedium;
+  justify-content: flex-end;
+  gap: $gapMedium;
 }
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: $gapBig;
+}
```

Comment:
we have Vertical container  for such purposes

Reply:
done